src/components/returns/OcOrderReturnItem.tsx

```diff
@@ -4,11 +4,11 @@ import {FunctionComponent} from "react"
 import {OrderReturnItem} from "ordercloud-javascript-sdk"
 import {priceHelper} from "../../utils/price.utils"
 
-interface OcOrderReturnItemListProps {
```

Comment:
You can remove the Oc prefix, but it should still be named OrderReturnItemListProps.

Consider removing the prefix from the filename as well.